.pre-commit-config.yaml

```diff
@@ -1,45 +1,66 @@
 ---
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.6.0
     hooks:
       - id: check-yaml
       - id: check-json
       - id: end-of-file-fixer
-        exclude_types: [json]
+        exclude_types:
+          - json
       - id: trailing-whitespace
       - id: no-commit-to-branch
       - id: debug-statements
       - id: requirements-txt-fixer
       - id: check-executables-have-shebangs
       - id: detect-private-key
+  - repo: local
+    hooks:
+      - id: update-type-stubs
+        name: Check for Type Stubs and Update Config
+        entry: ./update_precommit.py
+        language: system
+        files: ^requirements.*\.txt$
```

Comment:
Why do we need `files` argument? `update_precommit.py` has hard coded file names.

Reply:
This actually runs the update_precommit.py only when files matching this regex have been modified. By omitting the `files` argument the update_precommit.py hook would be executed on every commit.